tutorials/QuantumClassification/InsideQuantumClassifiers.ipynb

```diff
@@ -44,7 +44,7 @@
     "The steps 1-3 are repeated multiple times to estimate the probability of getting 0 or 1 in the final measurement.\n",
     "This probability is compared with a threshold (classifier bias) to produce the final classification result.\n",
     "\n",
-    "You can read more about the theory behind circuit-centric quantum classifiers in the [Microsoft QDK documentation](https://docs.microsoft.com/quantum/libraries/machine-learning/introduction) or in the [original paper](https://arxiv.org/abs/1804.00633). \n",
+    "You can read more about the theory behind circuit-centric quantum classifiers in the [Microsoft QDK documentation](https://docs.microsoft.com/azure/quantum/user-guide/libraries/machine-learning/introduction) or in the [original paper](https://arxiv.org/abs/1804.00633). \n",
```

Comment:
```suggestion
    "You can read more about the theory behind circuit-centric quantum classifiers in the [Microsoft QDK documentation](https://docs.microsoft.com/azure/quantum/user-guide/libraries/machine-learning/intro) or in the [original paper](https://arxiv.org/abs/1804.00633). \n",
```

Reply:
Nice catch, thank you! Fixing it in #609.